eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -293,10 +293,16 @@ function Set-ChangeLogContent {
 
 function Remove-EmptySections {
   param (
-    [Parameter(Mandatory = $true)]
+    $ChangeLogEntries,
```

Comment:
I'm not sure I like this extra logic here. We should instead create another helper function that Prepare-Release calls or we should use the SanitizeEntry option you added in your last PR.